telegraf.conf

```diff
@@ -0,0 +1,6898 @@
+# Telegraf Configuration
```

Comment:
Don't commit things that are unique to a particular dev or environment, like this config.